### ISSUE TITLE
Do not assume digital and analogue inputs are enabled

### ISSUE THREAD

**Issue by dajtxx** [DPIclimate/enviro-node#8]
Don't send values for digital and analogue inputs that are not in use.

**Comment by HarveyBates** [comment]
this for the kangaroo?

**Comment by dajtxx** [comment]
No, it was a general point made early on to say if possible the firmware shouldn't send say 4 values if there are 4 A2D ports, but only one of them is in use.

The ICT node sends values for all A2D & digital inputs and it's up to the decoder to know whether the values are useful or not.

I'm not sure how viable this is.
